packages/rocketchat-lib/server/methods/createToken.js

```diff
@@ -0,0 +1,13 @@
+Meteor.methods({
+	createToken({user}) {
```

Comment:
Change this parameter to only show which properties of the user you need. This way the clients calling this real-time method don't need to pass the full user object which in turn makes transmission over the wire faster.